### PR TITLE
Reformatting is_float_dtype()

### PR DESCRIPTION
**Pull request by Jsiewierski11**
Not sure if I commented the issue number in the right place.

### CODE REVIEW

**Comment by Jsiewierski11** [comment]
I was wondering if you could provide some feedback on some of the failed tests. The linting error is passing locally with pre-commit installed, is there a reason why it would be failing on the CLI?

**Comment by JLU-Neal** [comment]
Hey, sorry for the late response :D 
I took several days off and I will start reviewing it soon!

**Review by JLU-Neal — CHANGES_REQUESTED** [review]
Need examples here

File: ivy/container/data_type.py [hunk 1]
```diff
@@ -567,6 +567,35 @@ def is_float_dtype(
...
     ) -> ivy.Container:
+        """
+        `ivy.Container` instance method variant of `ivy.is_float_dtype`.
+        This method simply wraps the function,
+        and so the docstring for `ivy.is_float_dtype`
+        also applies to this method with minimal changes.
+
+        Parameters
+        ----------
+        key_chains : Union[List[str], Dict[str, str]]
```
Comment (JLU-Neal) [inline ivy/container/data_type.py:578]: The self parameter is needed here.
You should also add docstrings to the static function (I think it is above this instance method).

File: ivy/array/data_type.py [hunk 2]
```diff
@@ -104,6 +104,19 @@ def is_bool_dtype(self: ivy.Array) -> bool:
...
+        `ivy.Array` instance method variant of `ivy.is_float_dtype`. This method simply
+        checks to see if the array is of type `float`.
+
+        Parameters
+        ----------
+        None
+
+        Returns
+        -------
+        bool
```
Comment (JLU-Neal) [inline ivy/array/data_type.py:117]: out

File: ivy/container/data_type.py [hunk 3]
```diff
@@ -567,6 +567,35 @@ def is_float_dtype(
...
+            Boolean indicating whether to prune the
+            key-chains that were not applied. Default is False.
+
+        map_sequences : bool
+            Boolean indicating whether to map method
+            to sequences (list, tuple). Default is False.
+
+        Returns
+        -------
+        bool
```
Comment (JLU-Neal) [inline ivy/container/data_type.py:596]: out

File: ivy/functional/ivy/data_type.py [hunk 4]
```diff
@@ -1393,23 +1393,68 @@ def is_int_dtype(
...
+    Helper function to check if the input is a float or a float-like object.
+
+    Parameters
+    ----------
+    x : any
+        Input to check.
+
+    Returns
+    -------
+    bool
```
Comment (JLU-Neal) [inline ivy/functional/ivy/data_type.py:1407]: out

File: ivy/functional/ivy/data_type.py [hunk 5]
```diff
@@ -1393,23 +1393,68 @@ def is_int_dtype(
...
+        "True" if the input is a float or a float-like object, otherwise "False".
+    """
```
Comment (JLU-Neal) [inline ivy/functional/ivy/data_type.py:1409]: examples

File: ivy/functional/ivy/data_type.py [hunk 6]
```diff
@@ -1393,23 +1393,68 @@ def is_int_dtype(
...
     Parameters
     ----------
-    dtype_in
+    dtype_in : Union[ivy.Dtype, str, ivy.Array, ivy.NativeArray, Number]
         The array or data type to check
 
     Returns
     -------
-    ret
+    bool
```
Comment (JLU-Neal) [inline ivy/functional/ivy/data_type.py:1428]: ret should be kept.

**Review by JLU-Neal — CHANGES_REQUESTED** [review]
Please check the details of the review~
If you have any further questions, feel free to comment!

**Comment by Jsiewierski11** [comment]
Committed changes adding examples, ret back into the doc string, and out variables in the function arguments. There is a small conflict when I got replaced the "\" with an out argument.

**Comment by JLU-Neal** [comment]
> Committed changes adding examples, ret back into the doc string, and out variables in the function arguments. There is a small conflict when I got replaced the "" with an out argument.

Sorry, my bad. Just replace all the bool variables with ret.

File: ivy/container/data_type.py (hunk 1 above)
Comment (JLU-Neal) [reply ivy/container/data_type.py:583]: You should also add docstrings to the static function (I think it is above this instance method).

**Review by JLU-Neal — CHANGES_REQUESTED** [review]
Please resolve the remaining problems :D

**Comment by Jsiewierski11** [comment]
Looks like there is still a merge conflict with the / and the out variable. Is this something you have to fix on your end?

File: ivy/container/data_type.py (hunk 1 above)
Comment (Jsiewierski11) [reply ivy/container/data_type.py:710]: Both of these should have been resolved in the latest commit, no?

**Comment by JLU-Neal** [comment]
@Jsiewierski11 Thanks for your contribution to Ivy! Well done!

**Pull request merged**